devdoc/rc_ptr.md

```diff
@@ -0,0 +1,35 @@
+# `rc_ptr` requirements
+
+## Overview
+
+`rc_ptr` is a module that encapsulates reference-counted pointers.
+
+## Exposed API
+```c
+typedef void (*RC_PTR_FREE_FUNC)(void* context);
+
+typedef struct RC_PTR_TAG
+{
+    const void* ptr;
```

Comment:
>const [](http://example.com/codeflow?start=4&length=5)

likely this is not needed. `void*` can only be used meaningfully after dereference, and at the time of dereferencing it, the pointer type is known - and that pointer type should carry the qualifier rather than here.

Reply:
Removed